### DISCUSSION TITLE
左手側LEDの制御が効かない

### DISCUSSION THREAD

**Discussion by sora140016** [Yowkees/keyball#199] [Q&A]
はじめまして。
先月keyball44を購入させていただき、現在作成しております。
表題通り左手側LEDの色制御が効いてない状態です。

https://user-images.githubusercontent.com/126043756/220647821-4e868e38-5b57-4948-b0e0-ba5289510dce.MOV

トラックボール、キー入力は問題なく、本番ファームウェアも問題なく使えはするのですが、レイヤー3のRGBトグルをクリックすると右だけon/offでき左は変化しません。
またRGBトグル下のキーなども右は正常に動作し、左は正常に動作しないで早い点滅になったり…。といった現状です。何個かpromicroを用意し付け替えをしてはいるのですが同じような光り方をするので、左基盤側のLED回路がショートしているか？などと躓いております。もしアドバイス等いただければ幸いです。お忙しいところ申し訳ないですがよろしくお願いします。
![D0797539-1BDA-4925-9EED-AD6B01386C84](https://user-images.githubusercontent.com/126043756/220648208-80560858-7cc2-45f9-9380-abfd7efcda91.jpeg)

**Comment by sora140016** [comment]
自己解決しました。
LEDの付け方をしたら正常に動作するようになりました。
ありがとうございます。

**Comment by Yowkees** [comment]
天キーイベント等の対応で、気付くのが遅れまして大変申し訳ありませんでした。
ご自身で検討され、自己解決いただけたのことで安心しました。
以上、今後ともどうぞよろしくお願いいたします。
